src/types_.py

```diff
@@ -11,32 +11,29 @@
 
 
 @dataclasses.dataclass
-class MigrationInputs:
-    """Configurable parameters for migration mode.
+class UserInputs:
+    """Configurable user input values used to run upload-charm-docs.
 
     Attrs:
+        discourse_hostname: The discourse hostname.
```

Comment:
Do you think it would be worth while to describe this attribute a bit more? E.g., the base path to the discourse server?

Reply:
changed the description to `The base path(hostname) to the discourse server.`, also in `action.yaml`
Thank you for the feedback!